policyengine_us/variables/gov/states/me/tax/income/adjusted_gross_income/me_additions.py

```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class me_agi_additions(Variable):
+class me_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "ME AGI additions"
```

Comment:
```suggestion
    label = "Maine AGI additions"
```
While we are in here